tuned/plugins/hotplug.py

```diff
@@ -17,42 +17,68 @@ def cleanup(self):
 		self._hardware_events_cleanup()
 
 	def _hardware_events_init(self):
-		raise NotImplementedError()
+		pass
 
 	def _hardware_events_cleanup(self):
-		raise NotImplementedError()
+		pass
 
 	def _init_devices(self):
 		self._hardware_events_init()
 
 	def _hardware_events_callback(self, event, device):
 		if event == "add":
 			log.info("device '%s' added" % device.sys_name)
-			self._add_device(device)
+			self._add_device(device.sys_name)
 		elif event == "remove":
 			log.info("device '%s' removed" % device.sys_name)
-			self._remove_device(device)
+			self._remove_device(device.sys_name)
 
-	def _add_device(self, device):
-		device_name = device.sys_name
+	def _add_device_process(self, instance_name, instance, device_name):
+		log.info("instance %s: adding new device %s" % (instance_name, device_name))
```

Comment:
It would be nice to have a similar log message when removing the devices from a group